pkgs/development/libraries/libfprint-tod/default.nix

```diff
@@ -5,9 +5,11 @@
 
 # for the curious, "tod" means "Touch OEM Drivers" meaning it can load
 # external .so's.
-libfprint.overrideAttrs ({ postPatch ? "", mesonFlags ? [], ... }: let
-  version = "1.90.7+git20210222+tod1";
-in  {
+libfprint.overrideAttrs ({ postPatch ? "", mesonFlags ? [ ], ... }:
+let
+  version = "1.94.7+tod1";
```

Comment:
Quoting [my comment](https://github.com/NixOS/nixpkgs/pull/299624#discussion_r1542512340) from https://github.com/NixOS/nixpkgs/pull/299624 (seemingly duplicate of this PR)

> Newer versions of libfprint+tod have problems with enrolling fingerprints and upstream refuses to fix this issue, so we can never update beyond `1.90.7+git20210222+tod1`.
> 
> [#175212 (comment)](https://github.com/NixOS/nixpkgs/issues/175212#issuecomment-1175119104) https://gitlab.freedesktop.org/libfprint/libfprint/-/issues/484

Anyway, my laptop with Goodix fingerprint reader was recently stolen, so I no longer have any stake in this discussion.

Reply:
I just tested enrollment and it works for me:

```
~ ❯ sudo fprintd-delete ymatsiuk
found 1 devices
Device at /net/reactivated/Fprint/Device/0
Using device /net/reactivated/Fprint/Device/0
Fingerprints of user ymatsiuk deleted on Goodix Fingerprint Sensor 53xc
~ ❯ sudo fprintd-list ymatsiuk
found 1 devices
Device at /net/reactivated/Fprint/Device/0
Using device /net/reactivated/Fprint/Device/0
User ymatsiuk has no fingers enrolled for Goodix Fingerprint Sensor 53xc.
~ ❯ sudo fprintd-enroll ymatsiuk
Using device /net/reactivated/Fprint/Device/0
Enrolling right-index-finger finger.
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-retry-scan
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-stage-passed
Enroll result: enroll-completed
~ took 15s ❯ sudo fprintd-verify ymatsiuk
Using device /net/reactivated/Fprint/Device/0
Listing enrolled fingers:
 - #0: right-index-finger
Verify started!
Verifying: right-index-finger
Verify result: verify-match (done)
```

Reply:
@boozedog Could you please test this on your machine?

Reply:
> > Newer versions of libfprint+tod have problems with enrolling fingerprints and upstream refuses to fix this issue, so we can never update beyond `1.90.7+git20210222+tod1`.
> > [#175212 (comment)](https://github.com/NixOS/nixpkgs/issues/175212#issuecomment-1175119104) https://gitlab.freedesktop.org/libfprint/libfprint/-/issues/484

I took a closer look at these :point_up: 
IIRC we're using a [fork](https://gitlab.freedesktop.org/3v1n0/libfprint/-/blob/tod/README.tod.md) of libfprint for tod devices, so I do understand why [this](https://gitlab.freedesktop.org/libfprint/libfprint/-/issues/484) got closed (it was reported into the wrong repo, since libfprint never supported tod :shrug:). `libfprint-tod` package is the modified `libfprint` that supports external drivers and `fprintd-tod` is the upstream `fprintd` with `libfprint-tod` injected in place of `libfprint`. With all that said, I can't see any problems with bumping versions. Please correct me if I'm wrong :slightly_smiling_face:

Reply:
> @hmenke Are we still waiting for the confirmation from @boozedog ? I'm running this setup for 2 weeks successfully, I'm not pushing but rather curious how long should we wait before merging this; since longer we wait the more drift accumulates (there is already 1.94.7 available)?

Please don't wait on me for this one -- I no longer have relevant hardware for this issue!

Reply:
Thanks for the heads-up @boozedog :+1:

Reply:
As mentioned above, my laptop with Goodix fingerprint reader was stolen, so you may go ahead, but please be aware that others may hit the enrollment issue I quoted, so be prepared to roll it back.

Reply:
Please make sure to tag me in case I missed the issue report, like I mentioned earlier I'm running `Goodix Fingerprint Sensor 53xc` and have no issues, but I'm up for digging deeper if issues do occur :slightly_smiling_face: